pyproject.toml

```diff
@@ -1,3 +1,13 @@
+[build-system]
+requires = [
+  "pip >= 19.3.1",
```

Comment:
This looks unnecessary: why do you care about the pip version in an isolated build env? Nothing seems to call it.